cpu/mips32r2_common/lpm_arch.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2016, Imagination Technologies Limited and/or its
+ *                 affiliated group companies.
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include <mips/m32c0.h>
+#include "arch/lpm_arch.h"
+
+void lpm_arch_init(void)
+{
+
```

Comment:
I have implemented all that is possible for power management in the base mips32r2 architecture (ie execute a 'wait' instruction), anything else would be SoC specific.

I will delete these unimplemented functions.